app/assets/templates/layout/navbar_right.html.slim

```diff
@@ -1,12 +1,12 @@
 ul.nav.navbar-nav.navbar-right
   li.my-account
-    a(ng-click="goToMyAccount()") My Account
+    a(ng-click="goToMyAccount()")= I18n.t 'sprangular.my_account'
```

Comment:
If we translate this part on the server, changing locale would require reloading the entire app, should we try to data bind the translation locally?

`a(ng-click='' ng-bind=''My Account' | translate')`

Not sure how this effects perf (having so many ng-bind's)